src/afl-fuzz-mutators.c

```diff
@@ -26,27 +26,43 @@
 
 #include "afl-fuzz.h"
 
-void load_custom_mutator(afl_state_t *, const char *);
+struct custom_mutator * load_custom_mutator(afl_state_t *, const char *);
 #ifdef USE_PYTHON
 void load_custom_mutator_py(afl_state_t *, char *);
 #endif
 
+struct custom_mutator * load_mutator(afl_state_t *, const char *);
+
 void setup_custom_mutator(afl_state_t *afl) {
 
   /* Try mutator library first */
   u8 *fn = getenv("AFL_CUSTOM_MUTATOR_LIBRARY");
+  struct custom_mutator *mut; 
+  u8 count_of_mutators;
 
   if (fn) {
-
     if (afl->limit_time_sig)
       FATAL(
           "MOpt and custom mutator are mutually exclusive. We accept pull "
           "requests that integrates MOpt with the optional mutators "
           "(custom/radamsa/redquenn/...).");
 
-    load_custom_mutator(afl, fn);
+    u8 *fn_token = strsep(&fn, ";");
+
+    if (!fn_token) {
+      afl->mutator = load_custom_mutator(afl, fn);
+    }
+
+    while (fn_token)
+    {
+      mut = load_custom_mutator(afl, fn_token);
+      list_append(afl->list_mutators, mut);
+      fn_token = strsep(&fn, ";");
 
-    return;
+    }
+
+    mut = get_head(afl->list_mutators)->prev->data;
```

Comment:
Make use of the functions and defines in list.h (`list_append` and later `LIST_FOREACH`)